### ISSUE TITLE
Progressbar bars installation

### ISSUE THREAD

**Issue by zzj0402** [agermanidis/autosub#78]
Collecting progressbar>=2.3 (from autosub)
  Downloading progressbar-2.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\zzj04\AppData\Local\Temp\pip-build-m643aw0q\progressbar\setup.py", line 5, in <module>
        import progressbar
      File "C:\Users\zzj04\AppData\Local\Temp\pip-build-m643aw0q\progressbar\progressbar\__init__.py", line 59, in <module>
        from progressbar.widgets import *
      File "C:\Users\zzj04\AppData\Local\Temp\pip-build-m643aw0q\progressbar\progressbar\widgets.py", line 121, in <module>
        class FileTransferSpeed(Widget):
      File "c:\users\zzj04\appdata\local\programs\python\python37-32\lib\abc.py", line 133, in __new__
        cls = super().__new__(mcls, name, bases, namespace, **kwargs)
    ValueError: 'format' in __slots__ conflicts with class variable

**Comment by lennartq** [comment]
tl;dr: Install Python 2. It's not compatible with Python 3.

I just bumped into the same problem. [This question/answer](https://stackoverflow.com/questions/33063427/pip-install-fails-with-valueerror-format-in-slots-conflicts-with-class-v) says Progressbar is incompatible with Python 3. [The Wiki page](https://github.com/agermanidis/autosub/wiki) says to install Python 2.7, so I assume Autosub isn't meant to work with Python 3.

This could be mentioned in README.md. I'm not that familiar with Python stuff, so maybe it's usually assumed everything's for Python 2.
